runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsConfig.java

```diff
@@ -65,7 +65,15 @@ default boolean clue()
 	}
 
 	@ConfigItem(
-		position = 3,
+			position = 3,
+			keyName = "killcount",
+			name = "Killcount Command",
+			description = "Configures whether the Killcount command is enabled"
+	)
+	default boolean killcount() { return true; }
```

Comment:
Checkstyle is going to complain about this; it should be:

```
default boolean killcount()
{
    return true;
}
```

like it was in your initial commit.